engine/components-api/src/main/java/com/cloud/agent/AgentManager.java

```diff
@@ -152,4 +152,6 @@
     void notifyMonitorsOfHostAboutToBeRemoved(long hostId);
 
     void notifyMonitorsOfRemovedHost(long hostId, long clusterId);
+
```

Comment:
@Pearl1594 did you merge master? There are additional changes.

Reply:
@rhtyd I haven't merged master

Reply:
Can you check the diff, do we need changes around agent propagation?